cni/pkg/cmd/root.go

```diff
@@ -161,7 +174,11 @@ func init() {
 	registerBooleanParameter(constants.SkipTLSVerify, false, "Whether to use insecure TLS in kubeconfig file")
 	registerIntegerParameter(constants.MonitoringPort, 15014, "HTTP port to serve prometheus metrics")
 	registerStringParameter(constants.LogUDSAddress, "/var/run/istio-cni/log.sock", "The UDS server address which CNI plugin will copy log output to")
+	registerStringParameter(constants.CNIEventAddress, "/var/run/istio-cni/pluginevent.sock",
```

Comment:
This UDS socket was made configurable because the one above it (the log UDS socket) is too.

I don't have a strong opinion about whether it should be or not - if you do, happy to make it and the log UDS non-configurable in a followup.